ci-operator/config/openshift-priv/cluster-api-provider-kubevirt/openshift-priv-cluster-api-provider-kubevirt-main.yaml

```diff
@@ -172,8 +172,32 @@ tests:
       commands: |-
         set -x
         hypershift install
-        oc wait deployment -n hypershift operator --for=condition=Available --timeout=5m
-        oc annotate pod -n hypershift -l app=operator "hypershift.openshift.io/capi-provider-kubevirt-image=${CAPK_IMAGE}"
+        oc patch deployment -n hypershift operator --type=json -p '[{"op": "add", "path": "/spec/template/spec/containers/0/env/-", "value": {"name": "IMAGE_KUBEVIRT_CAPI_PROVIDER", "value": "'"${CAPK_IMAGE}"'"}}]'
+
+        # make sure the patch worked
+        oc get deployment -n hypershift operator -o yaml | grep "${CAPK_IMAGE}"
+
+        VALID_IMAGE="false"
+        # verify the image in the manager pod
+        for i in $(seq 20); do
+          sleep 60
+          # wait until the old pods is deleted; eventually we need only 2 pods to run
+          if [[ $(oc get pods -n hypershift -l "app=operator,hypershift.openshift.io/operator-component=operator,name=operator" --no-headers | wc -l) != 2 ]]; then
```

Comment:
we aren't guaranteed that these two pods are the most up to date.

Reply:
Fixed